modules/vf-graphql-holochain/queries/plan.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Top-level queries relating to Plan
+ *
+ * @package: HoloREA
+ * @since:   2019-05-27
+ */
+
+import { DNAIdMappings, injectTypename } from '../types'
+import { mapZomeFn } from '../connection'
+
+import {
+    Plan,
+    PlanConnection,
+} from '@valueflows/vf-graphql'
+
+export default (dnaConfig: DNAIdMappings, conductorUri: string) => {
+  const readOne = mapZomeFn(dnaConfig, conductorUri, 'planning', 'plan', 'get_plan')
+  const readAll = mapZomeFn(dnaConfig, conductorUri, 'planning', 'plan', 'get_all_plans')
+
+  return {
+    plan: injectTypename('Plan', async (root, args): Promise<Plan> => {
+      return (await readOne({ address: args.id })).plan
```

Comment:
Not sure if this is the moment, but noting that I'd like to make `mapZomeFn` take generic parameters so that we can define both the expected input type and the expected output type. 
That would mean we'd be able to have type checking and type safety on this line where we write `(...).plan` and it's so opaque why there is a property called `plan` on this data structure. 

e.g. it might look like `mapZomeFn<InputType, OutputType>(...)`

cc @pospi